src/cluster_legacy.c

```diff
@@ -1579,6 +1579,12 @@ clusterNode *clusterLookupNode(const char *name, int length) {
     return dictGetVal(de);
 }
 
+const char* clusterGetInternalSecret(size_t *len) {
+    // TODO: Change to the elected password.
+    *len = sizeof("bananalot") - 1;
+    return "bananalot";
```

Comment:
A static temporary secret, to be changed to the actual secret once it is added to the cluster (before merging of course).